CHANGELOG.md

```diff
@@ -38,9 +38,9 @@ All notable changes to this project will be documented in this file.
     - "2020.8"
     - "2021.6"
 - `ci-opencue`:
-    - "2019"
-    - "2020"
-    - "2021"
+    - "2019.10"
```

Comment:
Should this not be:
- 2018.10
- 2019.10
- 2020.10
- 2021.7

as in python/aswfdocker/data/versions.yaml ?

Reply:
Yes indeed, thanks JF!